validator/testcases/javascript/entity_values.py

```diff
@@ -149,6 +149,27 @@ def on_init(wrapper, arguments, traverser):
     return {'return': on_init}
 
 
+@register_entity('NewTabURL.override')
+def NewTabURL_override(traverser):
+    def on_override(wrapper, arguments, traverser):
+        from validator.testcases.javascript.predefinedentities import (
+            CUSTOMIZATION_API_HELP)
```

Comment:
Why the import here and not at the top of the file? If it's to avoid an import loop, please add a comment saying so